tests-builds-from-more-prs/README.md

```diff
@@ -0,0 +1,76 @@
+# Triggering testing farm tests with builds from other PRs (projects)
+
+Use case: having 2 builds for different PRs, being able to trigger TF tests using both builds with a Packit
+command comment in one PR
+
+## Architecture notes:
+
+`Copr allows interlinking Copr projects so that TF could just install proper packages and Copr would enable the whole dependency chain`
+
+- it is possible to specify `runtime_dependencies` in Copr: https://pagure.io/copr/copr/pull-request/2245#
+  (should be released soon) - list of external repositories that will be automatically enabled together with this project repository
+- we would need to have a dynamic way of specifying these `runtime_dependencies`
+- since we want concrete packages to be installed in TF (we pass the build ID and NVRs), this wouldn't be enough
+  - if users build all the PRs in one Copr repo (e.g.
+    [leapp](https://github.com/oamg/leapp/blob/351cfe75ce1a7e10d7e565d9fa0bdc946a9af66d/.packit.yaml#L9) and
+    [leapp-repository](https://github.com/oamg/leapp-repository/blob/8b228941d34b3fcf4322353806fdcafa9fbba550/.packit.yaml#L26) build their PRs in one common Copr repo)
+    we cannot enforce with this solution what should be installed during tests (build ID, NVRs).
+
+## Possible implementation
+
+### Requirements
+
+The requirement would be to have enabled Packit Copr build job on both repositories where the PRs are happening.
+Then, the testing farm test job could be triggered by comment specifying the other pull request ID
+and we would rely on builds being already done by Packit.
+
+### Implementation
+
+We should be able to get Copr build ID and package NVRs via our database having the other PR ID:
+`PullRequestModel.get_or_create(pr_id=xy, namespace=xy, repo_name=xy, project_url=xy).get_copr_builds()`
+and determine the latest build comparing one of the stored times (task_accepted, build_submitted).
+
+With having the Copr build ID and package NVRs of the other PR Copr build, we could just pass these
+into the Testing farm payload.
```

Comment:
This will initially work, 100%. I'm wondering if there will be a better way to do it long-term (can Copr help us? can we just pass the Copr project to TF and let it deal with it?).

As the implementation starts, we can experiment :)

Reply:
I asked Pavel about the sentence from the architecture notes and he proposed using `runtime_dependencies` which I mentioned in the doc, but with our current workflow where we specify the Copr builds in the TF payload directly via build ID and NVRs, this would not be enough. Also, I think the situation can change after resolving packit/packit-service/issues/1532.